datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -553,4 +564,12 @@ public AbstractKafkaBasedConnectorTask getConnectorTask() {
       return _task;
     }
   }
+
+  @Override
+  public List<BrooklinMetricInfo> getMetricInfos() {
```

Comment:
Since this is an abstract class, you need to fix all the extenders of this class to call super.getMetricsInfos() in their override of getMetricsInfos(), along with their existing code to return all metrics, otherwise this metric won't be picked up by them. Any in-house Connectors need to be fixed too.

Reply:
Done.